src/sentry/static/sentry/app/views/organizationGroupDetails/groupSimilarIssues/similarStackTrace/toolbar.tsx

```diff
@@ -69,12 +70,21 @@ class SimilarToolbar extends React.Component<Props, State> {
             <StyledToolbarHeader className="event-count-header">
               {t('Events')}
             </StyledToolbarHeader>
-            <StyledToolbarHeader className="event-similar-header">
-              {t('Exception')}
-            </StyledToolbarHeader>
-            <StyledToolbarHeader className="event-similar-header">
-              {t('Message')}
-            </StyledToolbarHeader>
+
+            {v2 ? (
+              <StyledToolbarHeader className="event-similar-header">
+                {t('Score')}
+              </StyledToolbarHeader>
+            ) : (
+              <React.Fragment>
+                <StyledToolbarHeader className="event-similar-header">
```

Comment:
Please are you sure this class exists?

Reply:
I copypasted it from the red parts of the diff. I don't see it used anywhere but I am not super interested in finding out if removing breaks anything...